ui/src/views/Tasks/NoTaskGroup/index.jsx

```diff
@@ -48,7 +48,12 @@ export default class NoTaskGroup extends Component {
       <Dashboard
         title="Task Groups"
         helpView={<HelpView description={description} />}
-        search={<Search onSubmit={this.handleTaskGroupSearchSubmit} />}>
+        search={
+          <Search
+            placeholder="Search Task Group ID"
+            onSubmit={this.handleTaskGroupSearchSubmit}
+          />
+        }>
```

Comment:
This will need a rebase on master.

Reply:
Cool. After @owlishDeveloper confirms this, I will make a change by rebasing it on the master branch.

Reply:
I do not agree with this change. The title says "Worker Pools" because it displays a list of worker pools. The route is `worker-manager` because it's the name of the service. I don't see the need to change anything so please don't land this! Thank you!

Reply:
Okay @owlishDeveloper.